Gemfile

```diff
@@ -2,7 +2,7 @@ source ENV['GEM_SOURCE'] || 'https://rubygems.org'
 
 gemspec
 
-if File.exists? "#{__FILE__}.local"
+if File.exist? "#{__FILE__}.local"
```

Comment:
as an alternative we could also delete this block

Reply:
I was a bit on the fence about that. Right now I'm going for easy